terraformInstall.sh

```diff
@@ -40,10 +40,26 @@ gpg --no-default-keyring \
     --fingerprint
 
 ## CONDITIONAL 4
-echo "Adding HashiCorp repository..."
-echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] \
+
+# Check if the HashiCorp repository is in the apt cache
+if ( apt-cache policy | grep "https://apt.releases.hashicorp.com" > /dev/null 2>&1 )
```

Comment:
You only need to check packages, not policy. If the source is added to the sources list, then you're ok.

Reply:
Do you mean that I don't need to check the cache first at all, or don't use the policy command?